File: filesets/language/Java/UntitledTest.java

import org.junit.*;
import static org.junit.Assert.*;

public class UntitledTest 
{
    @Test
    public void hitch_hiker()
    {
        int expected = 9 * 6;
        int actual = Untitled.answer();
        assertEquals(expected, actual);
    }
}

File: filesets/language/C#/UntitledTest.cs

using NUnit.Framework;

[TestFixture]
public class UntitledTest
{
    [Test]
    public void HitchHiker()
    {
        int expected = 6 * 9;
        int actual = Untitled.Answer;
        Assert.AreEqual(expected, actual);
    }
}

